test-data/unit/check-incremental.test

```diff
@@ -4283,3 +4283,31 @@ def foo() -> None:
 [out]
 [out2]
 tmp/main.py:2: error: "int" has no attribute "foo"
+
+[case testIncrementalBustedFineGrainedCache1]
+# flags: --cache-fine-grained
+import a
+import b
+[file a.py]
+[file b.py]
+-- This is a heinous hack, but we simulate having a invalid cache by clobbering
+-- the proto deps file with something with a hash mismatch.
+[file ../.mypy_cache/3.6/@proto_deps.meta.json.2]
+{"__main__": "00000000000000000000000000000000", "a": "d41d8cd98f00b204e9800998ecf8427e", "b": "d41d8cd98f00b204e9800998ecf8427e", "builtins": "3d22863e34b3d846306e1e76ebc82407"}
```

Comment:
I think we need to somehow exclude `builtins` from the snapshot in tests, otherwise all such tests need to be updated when we change something in `lib_stub`. Or alternatively use a dedicated named file and `[builtins ...]`.

Reply:
It doesn't matter in this case really because the point is that it is going to mismatch anyways, but this would be a problem in any situation where we want a correct hash.

I'll make the hash all 0s, though, to make clear it doesn't matter whether it matches.